client/js/models/OurOfferings.js

```diff
@@ -0,0 +1,17 @@
+module.exports = { ...require('./__proto__.js'),
+	data: [
+		{header: 'fall 2017', item: '', quantity: '', price: ''},
+		{header: '', item: 'Carrots', quantity: 'Bushel', price: '$ 1.50'},
+		{header: '', item: 'Kale', quantity: '1 lb.', price: '$ 3.00'},
+		{header: '', item: 'Eggs', quantity: 'A Dozen', price: '$ 6.25'},
+		{header: '', item: 'Magic Beans', quantity: 'Item Qty.', price: '$ 3.00'},
+		{header: 'year-round', item: '', quantity: '', price: ''},
```

Comment:
Also, avoid the blanks for the other properties. You can just explicitly state the `<h3>` in your html then have `<div>${fallOfferings}</div>`, etc., under it.